apps/teams-test-app/e2e-test-data/messageChannels.json

```diff
@@ -0,0 +1,26 @@
+{
+  "name": "Message Channels",
+  "checkIsSupported": {
+    "capabilityName": "MessageChannels",
+    "expectedOutput": "MessageChannels is not supported",
+    "testUrlParams": []
+  },
+  "platforms": "Web",
+  "testCases": [
+    {
+      "title": "getTelemetryPort API Call - Success",
```

Comment:
There should be a host SDK version restriction on this test. Let me find an example.

Reply:
Here's an example. I'm pretty sure the message channel feature went out in v2.12.0
"hostSdkVersion": {
        "web": ">=2.12.0"
 },

Reply:
Thank you! I forgot I was still trying to figure that bit out. Let me see if this fixes the build as well.

Reply:
One question, when is getTelemetryPort API is supported? You can keep version field but just make sure it is labelled correctly (which is to say, don't include a version of TJS (test app) that doesn't have that API

Reply:
Does the version in the test JSON represent the Test app version or the teams-js SDK version? I would have assumed the latter?

Reply:
You could understand it as both. They are tied to each other. You will never have a chance to run test app V2.20.0 on TJS V2.21.0 for example. The test pulls TJS V2.21.0 release branch and use test app under that branch's folder.

Reply:
Right, so with the fix for the test app in this PR, we can't run the capability test on isSupported until we can target the test app version with this UI feature.

I've updated the PR to remove the isSupported test block and we will need to add that back later.